tests/test_jsonnet.py

```diff
@@ -27,6 +29,15 @@ def test_yaml_dump(self):
         yaml = yaml_dump("{\"key\":\"value\"}")
         self.assertEqual(yaml, "key: value\n")
 
+    def test_parse_yaml(self):
+        """
+            This tests the yaml_to_json function.
+            It converts the yaml file in test_resources/ to a json string
+        """
+        current_pwd = os.path.dirname(__file__)
+        json = yaml_to_json([current_pwd], "test_resources/test_parse_yaml.yaml")
+        self.assertEqual(json, "{\"test\": {\"key\": \"value\", \"array\": [\"ele1\", \"ele2\"]}}")
```

Comment:
Can you also put ` "{\"test\": {\"key\": \"value\", \"array\": [\"ele1\", \"ele2\"]}}"` in an expected_json variable and then pass it to assertEqual? It makes the code self-explanatory and is a good practice.